.ci/jenkins/dsl/test.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -e
```

Comment:
Does any CI call this script?

Reply:
no CI nope. But pretty useful to test locally :)